src/ape_accounts/accounts.py

```diff
@@ -60,7 +59,7 @@ def keyfile(self) -> dict:
 
     @property
     def address(self) -> AddressType:
-        return to_address(self.keyfile["address"])
+        return to_checksum_address(self.keyfile["address"])
```

Comment:
A bigger TODO is to make `AccountAPI` know which ecosystem an account can work.

Then, we can use a non-connected version of the ecosystem to decode the address.
Otherwise if we do `self.provider.network.ecosystem.decode_address` here, it will require a provider to run the command `ape accounts list`.